src/__tests__/use-cases/delegates/manage-delegates.spec.ts

```diff
@@ -215,7 +215,7 @@ const buildSignature = async (
   const domain = { name: 'Safe Transaction Service', version: '1.0', chainId };
   const types = {
     Delegate: [
-      { name: 'delegateAddress', type: 'bytes32' },
+      { name: 'delegateAddress', type: 'address' },
```

Comment:
When changing to address you won't need the zero pad in the delegateAddress, check few lines below

https://github.com/safe-global/safe-core-sdk/pull/879/files

Reply:
Thanks @dasanra !! 🙂